File: Remove Duplicates from Sorted Array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        left=0
        i=1
        while i < len(nums):
            if nums[left]!=nums[i]:
                nums[left+1]=nums[i]
                left+=1
                i+=1
            else:
                i+=1
        return left+1

File: Java/Remove Duplicates from Sorted Array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int write = 1;
        int read;
        for ( read =1 ; read<nums.length;read++){
                if (nums[read]!=nums[write-1]){
                nums[write]=nums[read];
                write++;
                }
            }
        return write;     
    }
}
